cmd/root.go

```diff
@@ -79,6 +79,8 @@ var outputOptions = []string{
 	"csv",
 }
 
+var exitCode int
```

Comment:
Nit: We could probably move this up to the var block in line 65-ish to be a bit cleaner